addOns/retest/retest.gradle.kts

```diff
@@ -0,0 +1,35 @@
+description = "An extension to retest for presence/absence of previously generated alerts."
+
+zapAddOn {
+    addOnName.set("Retest")
+    zapVersion.set("2.10.0")
+
+    manifest {
+        author.set("ZAP Dev Team")
+        url.set("https://www.zaproxy.org/docs/desktop/addons/retest/")
+        extensions {
+            register("org.zaproxy.addon.retest.ExtensionRetest") {
+                classnames {
+                    allowed.set(listOf("org.zaproxy.addon.retest"))
+                }
+                dependencies {
+                    addOns {
+                        register("automation") {
+                            version.set(">=0.4.0")
+                        }
+                    }
+                }
```

Comment:
The whole add-on can depend directly on the add-on (rather than just the extension).